data/ntia/Coastal_DPAs_CatBNeighborhoods.csv

```diff
@@ -0,0 +1,41 @@
+Name,DPA Name,Neighborhood [km]
+East1,Norfolk,368
+East2,Jacksonville,464
+East3,Pascagoula,336
+East4,Maine,352
+East5,Boston,368
+East6,Nantucket,384
+East7,NewYork,368
+East8,Philadelphia,352
+East9,Buxton,416
+East10,CapeLookOut,368
+East11,Wilmington,368
+East12,Charleston,416
+East13,DaytonaBeach,336
+East14,PalmBay,336
+East15,WestPalmBeach,352
+East16,Miami,368
+East17,FloridaKeys,368
+East18,Naples,336
+East19,Sarasota,384
+East20,Tampa,352
+East21,Tallahassee,416
+East22,NewOrleans,368
+East23,BatonRouge,368
+East24,Houston,368
+East25,FreePort,352
+East26,CorpusChristi,352
+West14,SanDiego,192
+West13,LosAngles,160
+West12,SantaBarbara,224
+West11,SantaMaria,240
+West10,Monterey,192
+West9,SanFrancisco,144
+West8,Manchester,208
+West7,DelgadaCanyon,160
+West6,Eureka,160
+West5,PortOrford,176
+West4,Eugene,176
+West3,Portland,160
+West2,Aberdeen,192
+West1,LaPush,176

```

Comment:
In fact this is one of the open question which needs to be solved at the spec level, when updating the requirement to handle the variable catB distance.
I personally think that the new way of setting distance should render the high HAAT condition useless, but I guess this will be decided in WG1.
Meanwhile I have decided to keep the old behavior, and we will do a special PR to update this point. Do you agree?